### ISSUE TITLE
Add a personalization vector option for personalized pagerank?

### ISSUE THREAD

**Issue by ddu1** [neo4j-contrib/neo4j-graph-algorithms#720]
Maybe I missed it in the documentation, but I couldn't find an option to pass in a non-evenly-distributed personalization vector to the ppr algorithm, other than a list of `sourceNodes` (which implies evenly-distributed?) : 

https://github.com/neo4j-contrib/neo4j-graph-algorithms/blob/3.4.7.0/algo/src/main/java/org/neo4j/graphalgo/PageRankProc.java#L196

It would be a very nice feature to have, similar to what is implemented [here](https://networkx.github.io/documentation/networkx-1.10/reference/generated/networkx.algorithms.link_analysis.pagerank_alg.pagerank.html).

**Comment by rugsor** [comment]
I agree. I believe this is already discussed in #271 , where I suggest passing a named reference to a probability property on the nodes.
